app/templates/identity.hbs

```diff
@@ -25,7 +26,14 @@
                     <i class="fa fa-spinner fa-spin"></i>
                 {{/if}}
                 Save
-            </button> 
+            </button>
+            {{#if (eq this.profileURL this.model.profileURL)}}
+                <BsTooltip @title="This URL is already in use." @placement="right" />
+            {{else if  this.saveDisabled}}
+                <BsTooltip @title="Profile URL is empty!" @placement="right" />
+            {{/if}} 
```

Comment:
Please format these lines in terms of tab spacing

Reply:
formated the lines